docs/code/getting-started.md

```diff
@@ -675,4 +675,141 @@ Let’s recap what we learned in this section:
+                            <setting name="AuthPath">/o/oauth2/auth</setting>
+                            <setting name="TokenPath">/o/oauth2/token</setting>
+                            <setting type="int" name="UiPolicy">0</setting>
+                            <setting name="RedirectUri">http://elementary.io</setting>
+                            <setting name="ResponseType">code</setting>
+                            <setting name="Scope">https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/plus.login</setting>
+                            <setting name="AuthQuery">access_type=offline&amp;approval_prompt=force</setting>
+                            <setting name="ClientId">your-client-id</setting>
+                            <setting name="ClientSecret">your-client-secret</setting>
+                            <setting type="as" name="AllowedSchemes">['https','http']</setting>
+                            <setting type="as" name="AllowedRealms">['google.com']</setting>
+                            <setting type="as" name="Realms">['accounts.google.com']</setting>
+                        </group>
+                    </group>
+                </group>
+            </template>
+        </provider>
+
+Let's describe this file a little bit, at first all the files we are creating should be written in XML, the first field to set it the provider id, `<provider id="google">`. You have to make sure that this id is unique. Here are more informations about the keys:
+* The `icon` should be the icon name of the provider. Make sure to ship it along with the .provider file
+* The `translations` is the domain that should be installed and provide translations for the description
+* The `plugin` is the name of the plugin POA will use to handle the account
+
+As you can see, the biggest amount of lines are contained in the `template` key. They represent everything POA will need to authenticate, the required keys can variate between each methods and mechanisms so you might want to take a look at the keys that are supported by them.
+Nevertheless, a common part of the `template` is that you always have to specify the `auth/method` and `auth/mechanisms` settings, then you have to put all the supported keys in the `auth/$method/$mechanism` group. If any key is missing, the backend will fail and return you an explicit error message in the terminal.
+
+## Create a Service {#poa-create-service}
+
+As in the Create a Provider section, there are several default services shipped with elementary, and much more available in the AppCenter. If the service you want is already available, jump to the next part as you *MUST* use it. Otherwise, here is how to create a new one.
+The .service file should be stored in "/usr/share/accounts/services/", it is better to name the file as `%provider-%service.service` (here it’s `google-contacts.service` for example)
```

Comment:
Same question as the above file. Where do I store this in my project? How do I install it with CMake? Do I have to do anything with packaging?